cli/pkg/config_manager/config_manager.go

```diff
@@ -0,0 +1,450 @@
+			}
+			if len(happyNamespaces) == 0 {
+				logrus.Error("No happy namespaces were found in the selected cluster, please select a different region, profile or eks cluster.")
+				continue
+			}
+
+			defaultNamespace := happyNamespaces[0]
+			for _, namespace := range happyNamespaces {
+				if strings.Contains(namespace, fmt.Sprintf("-%s-happy-env", env)) {
+					defaultNamespace = namespace
+					break
+				}
+			}
+
+			prompt2 := &survey.Select{
+				Message: fmt.Sprintf("Which happy namespace should we use in %s?", env),
+				Options: happyNamespaces,
+				Default: defaultNamespace,
+			}
+
+			err = survey.AskOne(prompt2, &happyNamespace)
+			if err != nil {
+				return nil, errors.Wrapf(err, "failed to obtain an eks cluster id")
+			}
+			break
+		}
+
+		environments[env] = config.Environment{
+			AWSProfile: util.String(profile),
+			AWSRegion:  util.String(region),
+			K8S: k8s.K8SConfig{
+				Namespace:  happyNamespace,
+				ClusterID:  clusterId,
+				AuthMethod: "eks",
+			},
+			TerraformDirectory: fmt.Sprintf(".happy/terraform/envs/%s", env),
+			AutoRunMigrations:  false,
+			TaskLaunchType:     "k8s",
+		}
+	}
+
+	services := []Service{}
+	logrus.Info("We have found dockerfiles in your project, let's see if you'd like to use them as services in your stack")
+	for _, dockerPath := range dockerPaths {
+		dockerFileName := filepath.Base(dockerPath)
+		contextPath, err := filepath.Rel(bootstrapConfig.HappyProjectRoot, filepath.Dir(dockerPath))
+		if err != nil {
+			return nil, errors.Wrapf(err, "failed to obtain relative path")
+		}
+
+		confirm := false
+		prompt := &survey.Confirm{
+			Message: fmt.Sprintf("Would you like to use dockerfile %s/%s as a service in your stack?", contextPath, dockerFileName),
+		}
+		err = survey.AskOne(prompt, &confirm)
+		if err != nil {
+			return nil, errors.Wrap(err, "unable to prompt")
+		}
+		if !confirm {
+			continue
+		}
+
+		serviceName := ""
+		prompt1 := &survey.Input{
+			Message: fmt.Sprintf("What would you like to name the service for %s/%s?", contextPath, dockerFileName),
+			Help:    "This will be the name of the service in your stack, lowercased and hyphenated",
+			Default: filepath.Base(filepath.Dir(dockerPath)),
+		}
+		err = survey.AskOne(prompt1, &serviceName, survey.WithValidator(survey.Required), survey.WithValidator(survey.MinLength(3)))
+		if err != nil {
+			return nil, errors.Wrap(err, "unable to prompt")
+		}
+		serviceName = strings.ToLower(serviceName)
+
+		serviceType := "Service is exposed to the internet, and can only be consumed by other services in the stack (PRIVATE)"
+		prompt2 := &survey.Select{
+			Message: fmt.Sprintf("What kind of service is %s?", serviceName),
+			Options: []string{
+				"Serivce is exposed to the internet (EXTERNAL)",
+				"Service is exposed to the internet, but is protected by Okta (INTERNAL)",
+				"Service is exposed to the internet, and can only be consumed by other services in the stack (PRIVATE)",
```

Comment:
Can you replace this with serviceType or will that cause a weird cycle? Just trying to think how to not duplicate the string
```suggestion
				serviceType,
```